modules/openy_memberships_front/app/src/components/Product.vue

```diff
@@ -125,6 +125,8 @@ $vs-state-active-bg: #0060AF;
       font: 500 36px/54px 'Cachet', Verdana, sans-serif;
       letter-spacing: 0;
       color: #231F20;
+      margin: 0px;
```

Comment:
Please, move this styling to the component: 
https://github.com/ymcatwincities/openy_memberships/blob/master/scss/paragraph_skins/memberships-app.scss

Reply:
done